tests/models/test_lightfm.py

```diff
@@ -326,6 +326,30 @@ def test_second_fit_refits_model(self, dataset: Dataset) -> None:
         model = LightFMWrapperModel(model=base_model, epochs=5, num_threads=1)
         assert_second_fit_refits_model(model, dataset)
 
+    @pytest.mark.parametrize("use_features_in_dataset", (False, True))
```

Comment:
Could you add `parametrize` on losses? All except "warp-kos".

Reply:
Fixed [912cb6f](https://github.com/MobileTeleSystems/RecTools/pull/223/commits/912cb6f5442d7c0e597399e5647419944b8ca746)